src/ApiKey.env

```diff
@@ -1 +0,0 @@
-﻿export apiKey="COgbEuXTLZJ5nJkPgiJ68Uz7RaAcwp6z0GCrcoNj"
```

Comment:
Sad times! This key is now on the internet! :( 

So sadly it isn't secure any more. Lots of places have bots out and about that look for exactly this kind of mistake on GitHub!

I'm not going to worry about it too much this time (because its not really a very secret key) but on a real project we'd now need to destroy this key and make another one.